lang/en.json

```diff
@@ -198,6 +198,15 @@
   }
 },
 
+"DND5E.ACTIVEEFFECT": {
+  "FIELDS": {
+    "statuses": {
+      "label": "Status Conditions",
```

Comment:
In v13 the label disappeared unless I duplicated it here. Not sure why.

Reply:
Probably because it does not merge.

https://github.com/foundryvtt/foundryvtt/issues/11489

I should have remembered that.